README.md

```diff
@@ -14,6 +14,7 @@ replicated release create --auto -y
 
 ## Get started
 
+If you are new to writing Helm charts, check out examples of Helm chart written utilizing the Replicated Helm Library chart [here](https://github.com/replicatedhq/platform-examples)
 
 ### Copy and Clone
 
```

Comment:
(I can't comment lower down look a few lines below).

I think the line: 
`You should use the template to create a new **private** repo in your org, for example `mycompany/replicated-helm` or mycompany/replicated-starter-helm.`

Should be removed. I don't know why we would recommend our customers do that. Let's target the whole readme to a new customer who will see this in their collab repo.